File: MultiLanguageSandbox/src/test/deps/C#/46.cs
using System;
using System;
using System.Collections.Generic;
using System.Diagnostics;

class Program
{

/* Counts the number of characters in a given title string, excluding spaces and newline characters.
    >>> CountTitleCharacters("234")
    3
    >>> CountTitleCharacters("Ca 45")
    4
    >>> CountTitleCharacters("Hello World\n")
    10
    >>> CountTitleCharacters(" 123 ")
    3
*/

static int CountTitleCharacters(string title)
{
        int count = 0;
        foreach (char c in title)
        {
            if (c != ' ' && c != '\n')
            {
                count++;
            }
        }
        return count;
    }
    static void Main()
    {
        Debug.Assert(CountTitleCharacters("234") == 3);
        Debug.Assert(CountTitleCharacters("Ca 45") == 4);
        Debug.Assert(CountTitleCharacters("Hello World\n") == 10);
        Debug.Assert(CountTitleCharacters(" 123 ") == 3);
        Debug.Assert(CountTitleCharacters("\nNew\tLine") == 7);

    }
}

File: MultiLanguageSandbox/src/test/deps/php/46.php
<?php


/**
 * Calculates the number of characters in a composition's title.
 * The title may contain uppercase and lowercase English letters, digits, spaces, and newline characters.
 * The function counts the characters excluding spaces and newline characters.
 * 
 * Example usage:
 * - For a title "234", the function returns 3.
 * - For a title "Ca 45\n", including a newline character, the function returns 4.
 * 
 * @param string $title The title of the composition.
 * @return int The number of characters in the title, excluding spaces and newline characters.
 */

function countTitleCharacters(string $title): int {

    // Remove all spaces and newline characters from the title
    $cleanedTitle = str_replace([' ', "\n"], '', $title);
    // Return the length of the cleaned title
    return strlen($cleanedTitle);
}
function main(){
    assert(countTitleCharacters("234") === 3, 'Testcase 1 failed');
    assert(countTitleCharacters("Ca 45") === 4, 'Testcase 2 failed');
    assert(countTitleCharacters(" \nCa 45\n ") === 4, 'Testcase 3 failed');
    assert(countTitleCharacters("") === 0, 'Testcase 5 failed');
    
}


main();
?>